halo2_gadgets/src/utilities/lookup_range_check.rs

```diff
@@ -397,254 +468,304 @@ mod tests {
     };
     use pasta_curves::pallas;
 
-    use std::{convert::TryInto, marker::PhantomData};
+    use crate::{
+        sinsemilla::primitives::K,
+        tests::test_utils::test_against_stored_circuit,
+        utilities::lookup_range_check::{
+            LookupRangeCheck, PallasLookupRangeCheck, PallasLookupRangeCheckConfig,
+        },
+    };
 
-    #[test]
-    fn lookup_range_check() {
-        #[derive(Clone, Copy)]
-        struct MyCircuit<F: PrimeFieldBits> {
-            num_words: usize,
-            _marker: PhantomData<F>,
-        }
+    use std::{convert::TryInto, marker::PhantomData};
 
-        impl<F: PrimeFieldBits> Circuit<F> for MyCircuit<F> {
-            type Config = LookupRangeCheckConfig<F, K>;
-            type FloorPlanner = SimpleFloorPlanner;
+    #[derive(Clone, Copy)]
+    struct MyLookupCircuit<F: PrimeFieldBits, Lookup: LookupRangeCheck<F, K>> {
+        num_words: usize,
+        _field_marker: PhantomData<F>,
+        _lookup_marker: PhantomData<Lookup>,
```

Comment:
Can we have them both in one? `_marker: PhantomData<<F,Lookup>>` or similar?

Reply:
_marker: PhantomData<(F, Lookup)>